src/developer-guide/contributing/working-on-issues.md

```diff
@@ -1,6 +1,5 @@
 ---
 title: Working on Issues
-layout: doc
```

Comment:
This file seems to be missing some changes from https://github.com/eslint/eslint/compare/v8.25.0...v8.30.0